src/Connector.SqlServer/Utils/StringExtensions.cs

```diff
@@ -19,10 +19,38 @@ public static string ToSanitizedSqlName(this string value)
 
             if (char.IsDigit(sanitizedSqlName[0]))
             {
-                sanitizedSqlName = $"Table{sanitizedSqlName}";
+                sanitizedSqlName = $"_{sanitizedSqlName}";
+            }
+
+            if (sanitizedSqlName.Length > 127) // 127 instead of 128, since we need space for parameter declaration '@'
```

Comment:
I thought about doing that for values over 119. If they are larger than that, we should calculate hash for them.
In this case the chance of just having a vocalubary key with a collision is lower or statistically negligible.

With the current approach it's possible for people to mimic hash value by just having columns of length of 127